<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="dispFrame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="disp0Frame">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="disp0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="disp1Frame">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="disp1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="disp2Frame">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="disp2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="disp3Frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="disp3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="dispButtonFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>270</width>
       <height>60</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="disp0Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>15</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disp0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disp1Button">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>15</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disp1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disp2Button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>15</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disp2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disp3Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>15</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disp3</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="buttonFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>70</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="directButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Direct</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skipButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="turnFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="playP1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>P1 Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playP2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>P2 Play</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="pointsFrame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>320</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="p2Points2Frame">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p2Points2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="p2Points1Frame">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p2Points1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="p1Points2Frame_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p1Points2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="p1Points1Frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>60</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p1Points1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="p1PointsLabelFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p1PointsLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;P1 Points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="p2PointsLabelFrame_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="p2PointsLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;P2 Points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="rdFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>70</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="rdTextFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="rdLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;RD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="rdLabelFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>70</width>
       <height>100</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="rd">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>dispFrame</zorder>
   <zorder>buttonFrame</zorder>
   <zorder>turnFrame</zorder>
   <zorder>pointsFrame</zorder>
   <zorder>disp3</zorder>
   <zorder>disp2</zorder>
   <zorder>rdFrame</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
